lib/Models/Catalog/Ows/WebFeatureServiceCapabilities.ts

```diff
@@ -134,6 +134,41 @@ function getOutputTypes(json: any): string[] | undefined {
   return Array.isArray(outputTypes) ? outputTypes : [outputTypes];
 }
 
+interface SrsNamesForLayer {
+  layerName: string;
+  srsArray: string[]; // First element is DefaultSRS
+}
+
+/**
+ * Get the coordinate systems (srsName) supported by the WFS service for each layer.
+ * @param json
+ * returns an object with an array of srsNames for each layer. The first element is the defaultSRS as specified by the WFS service.
+ * TODO: For catalog items that specify which layer we are interested in, why build the array describing the srsNames for all the other layers too?
```

Comment:
Does this introduce inefficiency in the code?

Reply:
RE: "TODO: For catalog items that specify which layer we are interested in, why build the array describing the srsNames for all the other layers too?"

Reply:
I wouldn't worry. The main issue is that we have to load these `GetCapabilities` documents - which can be multiple MB - usually there is more overhead in load time than processing them in JS.

Reply:
Yeah its not a complex operation quick as. As long as its in the right place Im happy cheers